### PR TITLE
java

### PR DESCRIPTION
**Pull request by Cookie8421**
ScoreException can not be thrown,why? How to debug?

### CODE REVIEW

File: Custom ScoreException [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+		
+	}
+	public void score(int e) throws ScoreException{
+		if(e<0 || e>100){
+			throw new ScoreException(e);
+		}
+	}
+
+}
+public class ScoreException {
```
Comment (ashfaq-sb) [inline Custom ScoreException]: ScoreException class should extend Throwable Class like:
class ScoreException extends Throwable{
....}

**Review by ashfaq-sb — CHANGES_REQUESTED** [review]
Let me fix it

File: Custom ScoreException (hunk 1 above)
Comment (ashfaq-sb) [inline Custom ScoreException]: `class ScoreException extends Throwable{`
....`}`

File: Custom ScoreException [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+	public String warnMess(){
+		return message;
+	}
+
+}
+public class TestScore {
+	public static void main(String []args){
+		Score s1 = new Score();
+		try{
+			s1.score=-4;
```
Comment (ashfaq-sb) [inline Custom ScoreException]: `s1.score(-4);`

**Review by ashfaq-sb — APPROVED** [review]

File: Custom ScoreException (hunk 2 above)
Comment (Cookie8421) [reply Custom ScoreException]: appreciate you dude

Comment (ashfaq-sb) [reply Custom ScoreException]: Have you managed to correct it?

Comment (ashfaq-sb) [reply Custom ScoreException]: Have you managed to correct it?

**Comment by Cookie8421** [comment]
Yes
----- 原始邮件 -----
发件人：Muhammad Ashfaq <notifications@github.com>
收件人：Cookie0622/Cookie <Cookie@noreply.github.com>
抄送人：Cookie0622 <cookie_8421@sina.cn>,  Author <author@noreply.github.com>
主题：Re: [Cookie0622/Cookie] java (#1)
日期：2017年01月04日 01点45分

@ashfaq-sb commented on this pull request.



In Custom ScoreException:
> +}
+public class ScoreException {
+	String message;
+	public ScoreException(int s){
+		message = "Enter is invalid";
+	}
+	public String warnMess(){
+		return message;
+	}
+
+}
+public class TestScore {
+	public static void main(String []args){
+		Score s1 = new Score();
+		try{
+			s1.score=-4;

Have you managed to correct it?

&mdash;

**Pull request merged**